Sources/Orbit/Support/Layout/LazyVStackIfAvailable.swift

```diff
@@ -2,37 +2,23 @@ import SwiftUI
 
 struct LazyVStackIfAvailable<Content: View>: View {
```

Comment:
I don't think it's even a good idea to provide this specific wrapper at all. 🤔

Firstly, the naming isn't the luckiest part as _if available_ can be understood differently, be it even _if it's not available, don't show anything at all_. _PossiblyVStack` would at least reflect the actual result correctly, altho I'm not taking the copy for this name… 😄

Secondly, the pessimistic result might do more damage than benefits – consider someone attempting to render a collection of 1000 list items, having the hope of this component doing at least something a bit useful when laziness is not available not to dump the CPU completely. I'd rather let the consumers solve this explicitly in their separate codebase as the optimisation that could be considered would be f.e. paged content or something that will definitely not calculate and “draw” 1000 elements when there's no `LazyXStack` around.

Reply:
Thx for the idea, it hinted me to the right direction! Renamed simply to `LazyVStack`.

Reply:
…that still gives a false promise if the runtime platform doesn't match the minimum where the lazy counterpart is actually available. 🤔 What do we use this for exactly? I'd really prefer letting the end-user solve this issue on his own, considering different option, as we can't really say _hey, it's okay to drop some big collection on anything using this_ – that could probably only result to finding out it's not performing well and doing some `if-else`ing anyway.